scripts/2500/stage/demux-2500.bash

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+#   usage: demux-2500.bash <absolute-path-to-run-dir>
+#   The output i.e. Unaligned dir will be created 
+#   under $OUT_DIR
+
+set -eu -o pipefail
+
+shopt -s expand_aliases
+
+VERSION=5.4.2
+
+##########
+# PARAMS #
+##########
+
+IN_DIR=${1?'please provide a run dir'}
+OUT_DIR=${2?'please provide a demux dir'}
+EMAIL=barry.stokman@scilifelab.se
+
+RUN=$(basename ${IN_DIR})
+RUN_DIR=$(dirname ${IN_DIR})
+PROJECTLOG=${OUT_DIR}/${RUN}/projectlog.$(date +"%Y%m%d%H%M%S").log
+
+#SCRIPT_DIR=/home/barry.stokman/development/demultiplexing/scripts/2500/stage/  # use this when developing in a conda env
```

Comment:
I assume this should be removed, if the develop on this they should set up their own conda env

Reply:
fixed in https://github.com/Clinical-Genomics/demultiplexing/pull/144/commits/9bbfd86fcf3e6bcc930c39706aa00149ae3c664b